tests/test_translate.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+import unittest
+
+from pythainlp.translate import translate
+
+
+class TestTranslatePackage(unittest.TestCase):
+    def test_translate(self):
+        self.assertIsNotNone(translate("แมวกินปลา", source="th", target="en"))
+        self.assertIsNotNone(
+            translate(
+                "แมวกินปลา",
+                source="th",
+                target="en",
+                tokenizer="word"
+            )
+        )
+        self.assertIsNotNone(
+            translate(
+                "แมวกินปลา",
+                source="th",
+                target="en",
+                tokenizer="bpe"
+            )
+        )
+        self.assertIsNotNone(
+            translate(
+                "the cat eats fish.",
+                source="en",
+                target="th"
+            )
+        )
+        self.assertIsNotNone(
+            translate(
+                "the cat eats fish.",
+                source="en",
+                target="th",
+                tokenizer="bpe"
+            )
+        )
```

Comment:
how long does it take to run this test on CPUs?

Reply:
> 
> 
> how long does it take to run this test on CPUs?

It takes 3 - 5 minutes.